### ISSUE TITLE
Build error on latest flutter version

### ISSUE THREAD

**Issue by alexrainman** [amake/dynamic-fonts-flutter#2]
```
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/dynamic_fonts-2.0.0/lib/src/google_fonts_base.dart:15:1: Error: 'AssetManifest' is imported from both 'package:dynamic_fonts/src/asset_manifest.dart' and 'package:flutter/src/services/asset_manifest.dart'.
```

Any ideas how to solve it?

**Comment by wukgdu** [comment]
`AssetManifest` exists after flutter 3.10, so could rename `AssetManifest` in this repo as a workaround
I made it in https://github.com/wukgdu/dynamic-fonts-flutter

**Comment by amake** [comment]
This is addressed in v2.2.0
